Makefile

```diff
@@ -32,6 +31,8 @@ clean-test: ## remove test and coverage artifacts
 	rm -fr htmlcov/
 
 test :
+    python ./setup.py develop
```

Comment:
This make the makefile invalid. Makefile require tab, not space.
Also this will (try to) install the package in *system* (/usr/local).